app/directives/forms/edit/reservation.html

```diff
@@ -155,6 +155,23 @@ <h4>Options</h4></div>
                             </div>
                         </div>
                     </div>
+                    <div class="col-xs-4">
+                        <div class="form-group" style="margin-left:10px;margin-top:15px;">
+                            <div class="togglebutton">
+                                <label>
+                                    <input type="checkbox" ng-model="doc.video" ng-false-value="false" ng-false-value="true" ng-change="setVideoUrl(doc)">
+                                    <span ng-if="doc.video"><i class="fa fa-television" aria-hidden="true"></i> Video</span><span ng-if="!doc.video">No Video</span>
+                                </label>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-xs-8" ng-if="doc.video">
+                        <div class="form-group " style="margin-left:10px;margin-top:15px;" >
+                              <label class="control-label" for="title" style="margin: 0 0 0 0;">Url</label>
+                              <input class="form-control" id="title" type="url" ng-model="doc.videoUrl">
```

Comment:
Can you add a `placeholder="{{room.videoUrl || 'Please type the video url here' }}"`